src/universal/manifest.json

```
@@ -1,5 +1,5 @@
 {
-	"version": "2.6.0",
+	"version": "2.6.1",
```

```suggestion
	"version": "2.7.0",
```

Let's bump minor version, due to https://github.com/devcontainers/images/pull/864 ?